content_dev/docs/sdk/java/getting-started/clis.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: codelab
```

Comment:
If you want to use angle brackets literally, they must be either escaped, or be within a code block

Reply:
Good catch :)
f93a861c71331f789d5f21603c78a11b65039129